components/feature/addons/src/main/java/mozilla/components/feature/addons/amo/AddonCollectionProvider.kt

```diff
@@ -37,6 +37,8 @@ internal const val DEFAULT_COLLECTION_USER = "mozilla"
 internal const val DEFAULT_COLLECTION_NAME = "7e8d6dc651b54ab385fb8791bf9dac"
 internal const val COLLECTION_FILE_NAME_PREFIX = "mozilla_components_addon_collection"
 internal const val COLLECTION_FILE_NAME = "${COLLECTION_FILE_NAME_PREFIX}_%s.json"
+internal const val COLLECTION_FILE_NAME_WITH_LANGUAGE = "${COLLECTION_FILE_NAME_PREFIX}_%s_%s.json"
+internal const val REGEX_FILE_NAMES = "$COLLECTION_FILE_NAME_PREFIX(_\\w+)?_%s.json"
```

Comment:
@csadilek I updated the regex to support both files.

Reply:
👍